aptoide-database/src/main/java/cm/aptoide/pt/database/accessors/InstalledAccessor.java

```diff
@@ -1,162 +0,0 @@
-/*
- * Copyright (c) 2016.
- * Modified on 02/09/2016.
- */
-
-package cm.aptoide.pt.database.accessors;
-
-import androidx.annotation.NonNull;
-import cm.aptoide.pt.database.realm.Installation;
-import cm.aptoide.pt.database.realm.Installed;
-import cm.aptoide.pt.database.schedulers.RealmSchedulers;
-import io.realm.Sort;
-import java.util.List;
-import rx.Completable;
-import rx.Observable;
-import rx.schedulers.Schedulers;
-
-/**
- * Created on 01/09/16.
- */
-public class InstalledAccessor extends SimpleAccessor<Installed> {
-
-  private final InstallationAccessor installationAccessor;
-
-  public InstalledAccessor(Database db, InstallationAccessor installationAccessor) {
-    super(db, Installed.class);
-    this.installationAccessor = installationAccessor;
-  }
-
-  public Observable<List<Installed>> getAllInstalled() {
-    return database.getAll(Installed.class)
-        .flatMap(installs -> filterCompleted(installs));
-  }
-
-  /**
-   * @return all the entries from this table even not installed apps
-   * if you want only installed apps consider using the install manager
-   */
-  public Observable<List<Installed>> getAll() {
-    return database.getAll(Installed.class);
-  }
-
-  public Observable<List<Installed>> getAllInstalledSorted() {
-    return Observable.fromCallable(() -> database.get())
-        .flatMap(realm -> realm.where(Installed.class)
-            .findAllSorted(Installed.NAME, Sort.ASCENDING)
-            .asObservable()
-            .unsubscribeOn(RealmSchedulers.getScheduler()))
-        .filter(installed -> installed.isValid())
-        .flatMap(installed -> database.copyFromRealm(installed))
-        .subscribeOn(RealmSchedulers.getScheduler())
-        .observeOn(Schedulers.io())
-        .flatMap(installs -> filterCompleted(installs));
-  }
-
-  @NonNull private Observable<List<Installed>> filterCompleted(List<Installed> installs) {
-    return Observable.from(installs)
-        .filter(installed -> installed.getStatus() == Installed.STATUS_COMPLETED)
-        .toList();
-  }
-
-  public Completable remove(String packageName, int versionCode) {
-    return database.getRealm()
-        .doOnNext(realm -> database.deleteObject(realm, realm.where(Installed.class)
-            .equalTo(Installed.PACKAGE_NAME, packageName)
-            .equalTo(Installed.VERSION_CODE, versionCode)
-            .findFirst()))
```

Comment:
You dropped this findFirst(). I don't think this is a problem but there will be no side-effects right?